main.js

```diff
@@ -0,0 +1,17 @@
+const addButton = document.getElementById('add');
+const itemsContainer = document.getElementById('itemsContainer');
+const inputField = document.getElementById('inputField');
+
+addButton.addEventListener('click', () => {
+  const paragraph = document.createElement('p');
+  paragraph.classList.add('p-style');
+  paragraph.innerText = inputField.value;
+  itemsContainer.appendChild(paragraph);
+  inputField.value = '';
+  paragraph.addEventListener('click', () => {
+    paragraph.style.textDecoration = 'line-through';
+  });
+  paragraph.addEventListener('dblclick', () => {
+    itemsContainer.removeChild(paragraph);
+  });
+});
```

Comment:
- [ ] In the guidelines of this project, we require to add methods related to status updates ie (`completed`: `true` / `false`) in new JavaScript files and import them as modules. Please let's try it out.